x/projects/keeper/project.go

```diff
@@ -73,27 +73,55 @@ func (k Keeper) AddKeysToProject(ctx sdk.Context, projectID string, adminKey str
 	return k.projectsFS.AppendEntry(ctx, projectID, uint64(ctx.BlockHeight()), &project)
 }
 
-func (k Keeper) GetProjectDevelopersPolicy(ctx sdk.Context, developerKey string, blockHeight uint64) (policy types.Policy, err error) {
+func (k Keeper) GetProjectDevelopersPolicy(ctx sdk.Context, developerKey string, blockHeight uint64, adminPolicy bool) (policy types.Policy, err error) {
 	project, _, err := k.GetProjectForDeveloper(ctx, developerKey, blockHeight)
 	if err != nil {
 		return types.Policy{}, err
 	}
 
-	return project.Policy, nil
+	if adminPolicy {
+		return project.AdminPolicy, nil
+	}
+	return project.SubscriptionPolicy, nil
 }
 
-func (k Keeper) AddComputeUnitsToProject(ctx sdk.Context, developerKey string, blockHeight uint64, cu uint64) (err error) {
-	project, _, err := k.GetProjectForDeveloper(ctx, developerKey, blockHeight)
-	if err != nil {
-		return err
+func (k Keeper) AddComputeUnitsToProject(ctx sdk.Context, project *types.Project, cu uint64) (err error) {
```

Comment:
remove the pointer so we dont have to deal with nill

Reply:
done